tests/Doctrine/Tests/TestInit.php

```diff
@@ -16,9 +19,9 @@
 error_reporting(E_ALL | E_STRICT);
 
 // register silently failing autoloader
-spl_autoload_register(static function ($class) {
+spl_autoload_register(static function ($class) : bool {
```

Comment:
actually, the `return true` should be removed, and the return type should be `void`.

but this could be removed entirely by using `autoload-dev` in composer.json`

Reply:
I removed this. Thanks for the suggestion!